addons/hr_holidays/static/src/components/day_month_selection/day_month_selection.js

```
@@ -0,0 +1,31 @@
+import { registry } from "@web/core/registry";
+import { SelectionField, selectionField } from "@web/views/fields/selection/selection_field";
+
+export class DayMonthSelectionField extends SelectionField {
+    static props = {
+        ...SelectionField.props,
+        selectedMonth: { type: String, optional: true },
+    };
+
+
+    get options() {
+        const selectedMonth = this.props.record.data[this.props.selectedMonth];
```

If selectedMonth is undefined or invalid, this could fail silently.

---

added a condition to prompt the user and not give him any day options to select from